### PR TITLE
get names consistent; associated doc updates based on test walkthrough

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees the golang-ex changes for issue3358

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This example was copied from the official [beego samples](https://github.com/bee
 
 If you'd like to install it, follow [these instructions](https://github.com/openshift/golang-ex/blob/master/README.md#installation).  
 
-The steps in this document assume that you have access to an OpenShift deployment that you can deploy applications on.
+The steps in this document assume that you have access to an OpenShift deployment that you can deploy applications on, including the default set of ImageStreams defined.  Instructions for installing the default ImageStreams are available [here](http://docs.openshift.org/latest/admin_guide/install/first_steps.html).  If you are defining the set of ImageStreams now, remember to pass in the proper credentials via the --config option.
```
Comment (bparees) [inline README.md]: proper _cluster-admin_ credentials and to create the ImageStreams in the 'openshift' namespace

Comment (gabemontero) [inline README.md]: Done (here and where this same clause existed in the other README's.

**Comment by gabemontero** [comment]
@bparees golang-ex comments incorporated

**Comment by bparees** [comment]
lgtm.

**Pull request merged**